File: jekyun-park/week14/programmers/p12987.swift
//
//  main.swift
//  SwiftAlgorithms
//
//  Created by 박제균 on 2022/04/01.
//  Programmers > Summer-Winter Coding(~2018) > 숫자 게임

import Foundation

func solution(_ a: [Int], _ b: [Int]) -> Int {

    var answer = 0
    let numberOfPeople = a.count
    
    var a = a
    a.sort(by: >)
    var b = b
    b.sort(by: >)
    
    var j = 0
    for i in 0..<numberOfPeople {
        if b[j] > a[i] {
            j += 1
            answer += 1
        }
    }
    return answer
}

File: hyeonjun/14week/p12987.py
def solution(A, B):
    answer = 0
    length = len(A)
    A.sort()
    B.sort()
    index_A = 0
    index_B = 0
    while index_A < length and index_B < length:
        if A[index_A] < B[index_B]:
            index_A += 1
            answer += 1
        index_B += 1
    return answer
